wsdl/ver10/schema/onvif.xsd

```diff
@@ -1813,6 +1813,68 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 			</xs:extension>
 		</xs:complexContent>
 	</xs:complexType>
+
+
+	<!--===============================-->
+	<!--          FrequencyDecibelPair            -->
+	<!--===============================-->
+
+	<xs:complexType name="FrequencyDecibelPair">
+	<xs:sequence>
+		<xs:element name="CenterFrequency" type="xs:int">
+			<xs:annotation>
+				<xs:documentation>The center frequency of the band in hertz (Readonly). For example, value of 1000 refers to 1kHz and 2000 refers to 2kHz.</xs:documentation>
+			</xs:annotation>
+		</xs:element>
+		<xs:element name="Decibel" type="xs:float">
+			<xs:annotation>
+				<xs:documentation>Decibel value associated with the center frequency.</xs:documentation>
+			</xs:annotation>
+		</xs:element>
+		<xs:any namespace="##any" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>
+		<!-- first Vendor then ONVIF -->
+	</xs:sequence>
+	</xs:complexType>
+
+	<!--===============================-->
+	<!--          EQPreset            -->
+	<!--===============================-->
+	<xs:complexType name="EQPreset">
+		<xs:sequence>
+			<xs:element name="Token" type="tt:ReferenceToken">
+				<xs:annotation>
+					<xs:documentation>Unique identifier for the preset.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Name" type="xs:string">
+				<xs:annotation>
+					<xs:documentation>Preset name. (Readonly)</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="isDefault" type="xs:boolean">
+				<xs:annotation>
+					<xs:documentation>This shows if the EQ preset is set as the default. When the scheduler isn't active, this preset will be in use. Only one preset can be marked as default among the eqpresets linked to an audio output.</xs:documentation>
```

Comment:
Am I correctly understand that SetEQPreset with isDefault = true sets this preset default for **all** audio output listed in OutputTokensAvailable of the corresponding option?

Reply:
Your understanding is correct, when its set to default, its applicable for all audio output tokens(OutputTokensAvailable) listed in corresponding option.